docs/docs/pages/style-spec/root.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Root
+id: root
+description: Root level properties of a MapLibre style specify the map's layers, tile sources and other resources, and default values for the initial camera position when not specified elsewhere.
+contentType: specification
+order: 2
+layout: page
+hideFeedback: true
+products:
+- MapLibre Style Specification
+prependJs:
+    - "import Items from '../../components/style-spec/items';"
+    - "import ref from '../../../../tsc/src/reference/latest';"
```

Comment:
Can the docs import from the src but not from the tsc folder or this is a problem due to how the docs are deployed?